src/serac/numerics/equation_solver.hpp

```diff
@@ -17,9 +17,11 @@
 #include <variant>
 
 #include "mfem.hpp"
+#include "mfem/linalg/petsc.hpp"
```

Comment:
I think once https://github.com/LLNL/serac/pull/1138 merges, this line will no longer be needed?

Reply:
And where-ever else this file is included.

Reply:
actually, I think that they can already be removed -- I didn't realize that `mfem.hpp` already included `mfem/linalg/petsc.hpp`.